fonts/fonts.css

```diff
@@ -0,0 +1,8 @@
+  @font-face {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5575857)